### ISSUE TITLE
ChartLine cut off at min and max

### ISSUE THREAD

**Issue by joshuajung** [pichukov/LightChart#3]
Thanks a lot for this awesome library - it's a joy to use, and one of the few truly light and functional SwiftUI chart libraries out here. 👍 

One thing I noticed:
![IMG_04D40C478EC9-1](https://user-images.githubusercontent.com/2870684/108678221-172e9f00-74eb-11eb-97b5-08948a14f212.jpeg)

I assume this happens because the line stroke is equally applied to both sides of the line, and then cut off by frame. It would probably make sense to vertically move the chart inwards by half of the stroke width, so that the stroke is no longer cut off.

Thanks again!

**Comment by pichukov** [comment]
Thank you for a feedback and for raising this issue. The logic for chart stretching (respecting the stroke line width) to avoid cut-offs is already exists in the package. Looks like it's a more tricky bug. Unfortunately, I can't reproduce it. I was trying to make a chart similar to yours, but it doesn't have any cut-off for me. Can you please provide me more information on your example: OS Version, exact data set, all other params that you had there. It can help me to catch this bug.
Sorry for the delay with the response.

**Comment by joshuajung** [comment]
Thanks a lot for looking into this! I can reproduce this with this Swift UI View:

```swift
struct TotalReturnChartView: View {
    var points: [Double] = [0.0, 104.854958223384, 589.7754879743442, 563.6158279189177, 571.9942492738891, 610.4518755819294, 639.5078851570257, 643.56124424363, 648.9246337720185, 685.305585725806, 663.817501853284, 767.16962284946, 798.5999328516581, 767.0407367208675, 807.8775753581657, 812.6207342509123, 1303.8975286658965, 1342.4752670481844, 1319.9099543631949, 1329.5587940080698, 1407.3948054235664, 1425.3244344508216, 1494.5587336539477, 1477.9729983739485, 1515.0452730085888, 1919.4849539887412, 1711.3097312439068, 1725.0809166285635, 1684.5154147787046, 1613.1691688185183, 1647.037773127827, 1592.5804111910095, 2243.0662812957885, 2149.1491334649586, 2253.763546493726, 2156.8164631974028, 2216.555193596682, 2233.533967277863, 2278.730304726056]
    var body: some View {
        LightChartView(data: points, type: .curved, visualType: .filled(color: .accentColor, lineWidth: 10))
            .padding(.vertical, 10)
            .frame(height: 100)
    }
}
```

<img width="553" alt="Bildschirmfoto 2021-03-07 um 17 18 02" src="https://user-images.githubusercontent.com/2870684/110246745-1d039600-7f69-11eb-8165-f4305f499e56.png">

This seems to apply to all devices I tested on in the XCode 12.4 sim. Please let me know if there is anything else I can help with.

**Comment by pichukov** [comment]
Thanks for it, I found a bug, it should be fixed in `1.0.2` version. Check this new version on your side and ping me if you still have some issues.

**Comment by joshuajung** [comment]
Awesome, that has done the trick. Thank you!
